MegaApiClient/MegaApiClient.cs

```diff
@@ -574,7 +574,10 @@ public Stream Download(INode node, CancellationToken? cancellationToken = null)
       DownloadUrlRequest downloadRequest = new DownloadUrlRequest(node);
       DownloadUrlResponse downloadResponse = this.Request<DownloadUrlResponse>(downloadRequest);
 
-      Stream dataStream = new BufferedStream(this.webClient.GetRequestRaw(new Uri(downloadResponse.Url)));

```

Comment:
The BufferedStream was introduced for performance reasons. I think you should keep it for now.
I think it could be
```csharp
Stream dataStream = new BufferedStream(new SeekableStream([...]));
```